### PR TITLE
Require vs Modifier

### PR DESCRIPTION
**Pull request by bidyut-arianelabs**
In this PR, I am replacing the functions with require to `modifier`.

### CODE REVIEW

**Review by cliffhall — APPROVED** [review]
LGTM! 👍

**Comment by mischat** [comment]
LGTM2 :)

**Review by mischat — APPROVED** [review]
Let's not merge this until after the split, thanks

**Pull request merged**
